### ISSUE TITLE
Improve sound/music playback timing

### ISSUE THREAD

**Issue by NY00123** [fragglet/sdl-sopwith#24]
Right now, the title music plays slower than the original matching DOS version.

This experimental change partially resolves the differences, but there may still be precision errors (especially right after starting the program):

```
diff --git a/src/swsound.c b/src/swsound.c
index 1da8869..d3d68b9 100644
--- a/src/swsound.c
+++ b/src/swsound.c
@@ -607,7 +607,7 @@ void swsndupdate(void)
        int thisclock = Timer_GetMS();
 
        if (thisclock > lastclock + 1000 / 18.2) {
-               lastclock = thisclock;
+               lastclock += 1000 / 18.2;
                soundadj();
        }
 }
```

**Comment by NY00123** [comment]
Reopening this, because my patch has a problem that I recall observing and is still reproduced here: A part of the track is skipped upon starting the program, but not later (after "game over").

**Comment by fragglet** [comment]
I think I fixed this in 97d4b004ece5a3b822b568ddc5d1974bf3f7e43b - can you confirm?

**Comment by NY00123** [comment]
Seems to be good, thanks!
